expan/core/version.py

```diff
@@ -1,6 +1,6 @@
 import subprocess
 
-__version__ = "1.0.1"
+__version__ = "1.1.0"
```

Comment:
ah you don't do it here. let's do it via bumpversion: http://expan.readthedocs.io/en/latest/contributing.html#release

Reply:
@shansfolder ok, got it!